content/docs/config/fallback/content.md

```diff
@@ -0,0 +1,46 @@
++++
+fragment = "content"
+weight = 101
+
+title = "Fallback"
+
+[sidebar]
+  enable = true
+  sticky = true
++++
+
+## Flow
+
+---
+
+Step-by-Step fallback flow:
```

Comment:
In case of path chaining, if the second record triggers the fallback, we shouldn't use the parent record's to= field as the first option like the current record's fallback flow because the parent record's to= field is pointed to the second record which triggered fallback.